src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -69,9 +82,15 @@ public function supportsNormalization($data, $format = null)
     public function denormalize($data, $class, $format = null, array $context = array())
     {
         $dateTimeFormat = isset($context[self::FORMAT_KEY]) ? $context[self::FORMAT_KEY] : null;
+        $timezone = $this->getTimezone($context);
 
         if (null !== $dateTimeFormat) {
-            $object = \DateTime::class === $class ? \DateTime::createFromFormat($dateTimeFormat, $data) : \DateTimeImmutable::createFromFormat($dateTimeFormat, $data);
+            if (null === $timezone) {
+                // https://bugs.php.net/bug.php?id=68669
```

Comment:
We can remove this workaround in Symfony 4.0, but for now the Symfony 3 [still supports PHP 5.5](https://github.com/symfony/symfony/blob/master/composer.json#L19).
See failing tests on Travis: https://travis-ci.org/symfony/symfony/jobs/222333390#L2437-L2471